pyfahviewer/static/main.js

```diff
@@ -40,8 +44,14 @@
   }
 
   async function refreshSlots() {
-    let response = await fetch(`${host}/api/slots`);
-    if (!response.ok) {
+    let response = null;
+    try {
+      response = await fetch("/api/slots");
+    } catch (e) {
+      console.log("Error when fetching slots: " + e);
```

Comment:
Suggestion: Using `console.error` and passing the error as an argument instead of `console.log`/concatenation will make it stand out in the console more and also retain other info like the stack trace:

![image](https://user-images.githubusercontent.com/2646487/79671334-96e73100-8197-11ea-86e1-b2608f50d336.png)